pkgs/games/BeatSaberModManager/default.nix

```diff
@@ -47,6 +49,11 @@ buildDotnetModule rec {
     fontconfig
   ];
 
+  # Required for OneClick
+  makeWrapperArgs = [
+    ''--prefix PATH : "${lib.makeBinPath [ xdg-utils ]}"''
```

Comment:
Please make this a `--suffix` because we want `xdg-open` to be available somehow but we don't want to override what is already in the environment.

Reply:
Well, it's too late now, I merged one minute before you commented D:

I don't think overriding the environment's xdg-utils is too bad since it's a very narrow scope. Also, we can't know whether the environment's xdg-utils works or not while we can be pretty sure ours does.

Reply:
By prefixing or suffixing the path we know that xdg-open will be available, but in either case we can't be sure that it works or not, because that depends on the runtime configuration. There is no guarantee what the behavior of xdg-open is or if there is an appropriate handle available at runtime.

https://github.com/NixOS/nixpkgs/pull/181171
https://discourse.nixos.org/t/should-we-support-encoding-runtime-dependencies-without-requiring-a-specific-implementation/21031/5